src/validation-messages/failures/invalid-response.ts

```diff
@@ -0,0 +1,9 @@
+import ValidationFailure from './validation-failure';
+
+class InvalidResponse extends ValidationFailure {
+  constructor(path) {
```

Comment:
Seems like the path isn't really necessary for this message. Would it make sense to have it defaulted to an empty array in the parent's constructor so that failures that don't really require a path don't have to pass in an empty array?